lib/gettext/po_entry.rb

```diff
@@ -428,11 +428,12 @@ def format_reference_comment
       end
 
       def format_flag_comment
-        formatted_flags = String.new
-        @entry.flags.each do |flag|
-          formatted_flags << format_comment(FLAG_MARK, flag)
+        if @entry.flags.empty?
+          return String.new
```

Comment:
Remove needless `return`:

```suggestion
          String.new
```